modular_robot_simulation/revolve2/modular_robot_simulation/build_multi_body_systems/_builder_params.py

```diff
@@ -0,0 +1,14 @@
+from dataclasses import dataclass
+
+from revolve2.simulation.scene import JointHinge, MultiBodySystem, Pose, RigidBody
+
+from .._modular_robot_active_hinge_key import ModularRobotActiveHingeKey
+
+
+@dataclass
+class BuilderParams:
+    """Parameters for the Builder to build a new Module."""
+
+    params: tuple[
```

Comment:
But can you unpack this tuple in to separate variables xD then I can better understand what each tuple index means. so basically don't use a tuple